package.json

```diff
@@ -114,44 +114,44 @@
     "inputmask": "^5.0.5",
     "prismjs": "1.27.0",
     "reflect-metadata": "0.1.8",
-    "rxjs": "7.5.6",
+    "rxjs": "~7.5.0",
```

Comment:
I'm a bit confused as to why this is changed. Should we revert to pointing at 7.5.6 specifically?

Reply:
Updating to patched versions og 7.5 should be fine.

Reply:
I agree, but it is not really something we have done for our other dependencies, at least not in any systematic way, so I would personally prefer to just pin it to a specific version.

Reply:
np; done